Don't require sqlite3 when using postgres

anoadragon453 opened the pull request:
There's no need to import and use sqlite3 if you're using postgres as your db backend.

richvdh commented on changelog.d/4466.misc:
```diff
@@ -0,0 +1 @@
+Don't require sqlite3 when using postgres as the database backend.
```
this isn't a change since the last release, so doesn't make sense to list in the changelog. (If it *was* a change since the last release, it would be a bugfix, not a misc.)

It would make more sense to consider it as part of the PR which introduced the breakage (#4306, it turns out). So copy 4306.misc to 4466.misc, and the two will share a line when towncrier builds the changelog.

anoadragon453 commented:
Ah, good to know, thanks!

On January 24, 2019 5:43:22 PM UTC, Richard van der Hoff <notifications@github.com> wrote:
>richvdh requested changes on this pull request.
>
>lgtm apart from the changelog
>
>> @@ -0,0 +1 @@
>+Don't require sqlite3 when using postgres as the database backend.
>
>this isn't a change since the last release, so doesn't make sense to
>list in the changelog. (If it *was* a change since the last release, it
>would be a bugfix, not a misc.)
>
>It would make more sense to consider it as part of the PR which
>introduced the breakage (#4306, it turns out). So copy 4306.misc to
>4466.misc, and the two will share a line when towncrier builds the
>changelog.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/matrix-org/synapse/pull/4466#pullrequestreview-196163442
